cibyl/sources/elasticsearch/api.py

```diff
@@ -470,6 +486,30 @@ def get_tests(self, **kwargs):
 
         return job_builds_found
 
+    def match_filter_test_by_duration(self: object,
+                                      test_duration: float,
+                                      test_duration_arguments: list) -> bool:
+        """Match if the duration of a test pass all the
+        conditions provided by the user that are located
+        in the arguments
+
+        :params test_duration: Duration of a job
+        :type node_name: float
+        :params test_duration_arguments: Conditions provided
+        by the user
+        :type list: str
+
+        :returns: Return if match all the conditions or no
+        :rtype: bool
+        """
+        for test_duration_argument in test_duration_arguments:
+            operator = RANGE_OPERATORS[test_duration_argument.operator]
+            operand = test_duration_argument.operand
+            if operator(test_duration, float(operand)):
```

Comment:
Nit: you can invert the condition here and simply return False if met, so it would look like:
```python
if not operator(test_duration, float(operand)):
    return False
```

Reply:
Oh, I didn't realize it. You're totally right. It doesn't make sense to continue checking if it match the condition or not once one doesn't do it.

Done.